game/world/objects/npc.cpp

```diff
@@ -2572,22 +2556,32 @@ void Npc::emitSoundEffect(std::string_view sound, float range, bool freeSlot) {
   sfx.play();
   }
 
+static const char* MaterialGroupNames[] = {
```

Comment:
should this be moved to `phenix`?

Reply:
Hm I don't think so. If that's important to you I can do it but it isn't something that belongs in `phoenix`, really.